copier.yaml

```diff
@@ -28,8 +29,8 @@ _tasks:
   - "npx json -If apps/ontodia/package.json 
       -e 'this.name=\"@{{app_name}}/explorer\"'
     "
-  - "sh .copier/generate_apis.sh"
-  - "rm -rf .copier"
+  - "python3 .copier/_generate_apis.py || python .copier/_generate_apis.py"
+  - "python3 _delete_copier_dir_and_self.py || python _delete_copier_dir_and_self.py"
```

Comment:
Thinking out loud here, since copier is already a python process, is this possible to run the `.py` without going to the shell?

This is what the [docs](https://copier.readthedocs.io/en/latest/configuring/#tasks) mention. Ah, I think the last example task is exactly that. I missed that before

```
_tasks:
    # Your script can be run by the same python environment used to run copier
    - ["{{ _copier_python }}", task.py]
```

Reply:
![image](https://user-images.githubusercontent.com/23079007/161316928-d82528c8-60d0-4fa0-937c-1da66dbc44a6.png)
Hmm ... this for me doesn't provide the hoped path to the right python runtime/interpreter -> it gives some random `'', '` in place of the `{{ _copier_python }}` :/

Reply:
@tpluscode when you have a moment I wanna know what if `{{ _copier_python }}` runs on your system  - no luck for me :/ - I'd suggest if it doesn't work as documented in Copier docs, we stick to what's right now _(and merge, if this is the only thing left unresolved 🤞)_

Reply:
I haven't checked but maybe be it's an unreleased change?

I opened [this discussion](https://github.com/copier-org/copier/discussions/639). I'll let you know. Do subscribe to get notified of the answer.

Reply:
ha! Today's release introduces it haha, literally fresh-from-the-oven stuff ;) 
Just checked, it works 💪  [289a5e4](https://github.com/hypermedia-app/creta-template-app/pull/30/commits/289a5e4b63262dc38220c46039529fc87539145f).
One thing I need to further understand is what Python version it actually uses -> I'm inside venv so I would expect the path to python executable be "venv/bin/python" - however, I get `/usr/bin/python3` but actually uniformly for both what copier uses and what's when I run `which python`: 
![image](https://user-images.githubusercontent.com/23079007/163361570-92f5d308-f70d-4b28-bc13-da71c1eb3ce8.png). 
Edit: I think my virtualenv on WSL is screwed up -> I enter into it and it still uses my system-level python interpreter eh ... -> but in any case this resolves the issue why `{{ _copier_python }}` uses the system-wide python for me also ;p  @tpluscode when you have a moment please have a go on your system out of curiosity :) .

I suggest we can merge this now with the newest change 👍 :)

Reply:
I got

>  > Running task 10 of 10: ['/Users/tomaszpluskiewicz/.local/pipx/venvs/copier/bin/python', '_delete_copier_dir_and_self.py']